src/Compilers/CSharp/Portable/Binder/Binder_Conversions.cs

```diff
@@ -1119,6 +1120,8 @@ static ErrorCode getRefMismatchErrorCode(TypeKind type)
                 return ReportUnsafeIfNotAllowed(syntax, diagnostics);
             }
 
+            ReportDiagnosticsIfUnmanagedCallersOnly(diagnostics, selectedMethod, location);
```

Comment:
Seems like we should check whether an error was produced or not.
Not sure how to deal with lazy diagnostic situation though...

Reply:
We can't, unfortunately, without causing a binding cycle. All returning true will do is suppress cascading warnings/errors, so I don't think it's of very high importance.

---
In reply to: [481405335](https://github.com/dotnet/roslyn/pull/47311#discussion_r481405335) [](ancestors = 481405335)